site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/MenuBuilder.java

```diff
@@ -117,7 +120,8 @@ public static void buildMenuForMembership( VelocityPortlet portlet, RunData data
         menu.add( buildMenuEntry( rl.getString( "mb.cursit" ), "doGoto_unjoinable", activeTab.equals( MembershipActiveTab.CURRENT_SITES ) ) );
 
         // Official course enrolments
-        menu.add( buildMenuEntry( rl.getString( "mb.enrolments"), "doGoto_enrolments", activeTab.equals( MembershipActiveTab.OFFICIAL_ENROLMENTS ) ) );
+        boolean courseManagementEnabled = ServerConfigurationService.getBoolean( SAK_PROP_CM_IMPLEMENTED, SAK_PROP_CM_IMPLEMENTED_DEFAULT );
+        if (courseManagementEnabled) menu.add( buildMenuEntry( rl.getString( "mb.enrolments"), "doGoto_enrolments", activeTab.equals( MembershipActiveTab.OFFICIAL_ENROLMENTS ) ) );
```

Comment:
```suggestion
        if (courseManagementEnabled)
        {
                menu.add( buildMenuEntry( rl.getString( "mb.enrolments"), "doGoto_enrolments", activeTab.equals( MembershipActiveTab.OFFICIAL_ENROLMENTS ) ) );
        }
```

Reply:
OK, thanks Bryan, I have done it and squashed the two commits.